PIL/ImageMode.py

```diff
@@ -40,6 +40,7 @@ def getmode(mode):
             _modes[m] = ModeDescriptor(m, bands, basemode, basetype)
         # extra experimental modes
         _modes["LA"] = ModeDescriptor("LA", ("L", "A"), "L", "L")
+        _modes["La"] = ModeDescriptor("La", ("L", "a"), "L", "L")
```

Comment:
This required for `Image.merge('La', ...)`. But there is no 'RGBa', so I used 'RGBX', which is. Modes support in Pillow is a mess ((